src/components/Requirements/RequirementDebugger.vue

```diff
@@ -54,11 +65,11 @@ function computeRequirementGraphForDisplay(
   return { requirements, courses, edges };
 }
 
-type Data = { onlyNodeToKeepEdge: string | number | null };
+type Data = { onlyNodeToKeepEdge: string | number | null; graphType: 'safe' | 'dangerous' };
```

Comment:
Is there a reason we use a union type here as opposed to using an enumerated type / boolean?

Reply:
Mostly to make the intention clear, and be consistent in naming with the rest of the system